src/gql/generated/types.ts

```diff
@@ -3909,6 +3909,15 @@ export type GetUserConfigQuery = {
   }>;
 };
 
+export type GetUserPermissionsQueryVariables = Exact<{ [key: string]: never }>;
+
+export type GetUserPermissionsQuery = {
+  user: {
+    userId: string;
+    permissions: { canCreateProject: boolean; userId: string };
```

Comment:
Looks like `yarn codegen` needs to be run again; this `userId` field was removed from the query.